executors/clipseg/executor.py

```diff
@@ -0,0 +1,214 @@
+import enum
+import os
+import sys
+import shutil
+import time
+
+from PIL import Image, ImageOps
+from io import BytesIO
+from pathlib import Path
+from typing import Dict
+from urllib.request import urlopen
+
+import cv2
+import numpy as np
+import torch
+
+from jina import Executor, DocumentArray, Document, requests
+from torchvision import transforms
+
+
+class THRESHOLDING_METHODS(str, enum.Enum):
+    NONE = 'none' # Do not threshold
+    BINARY = 'binary'
+    ADAPTIVE_MEAN = 'adaptive_mean'
+    ADAPTIVE_GAUSSIAN = 'adaptive_gaussian'
+
+
+DALLE_CACHE_DOCKER_DIR = '/home/dalle/.cache'
```

Comment:
Why is there some dalle stuff here ?

The code of each Executor should be independent. Maybe you should pass this cache folder in the init method of the Executor

Reply:
It was for the root level docker to have somewhere to put the weights, since that's the recommended mount point of the `.cache` folder. I can put it into the config for the folder if that is better (config.yml).